actions.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"errors"
+	"github.com/Terag/kadok/security"
+	"github.com/bwmarrin/discordgo"
+	"strings"
+)
+
+// ExecuteAction is the function type to implement with an Action
+type ExecuteAction func(s *discordgo.Session, m *discordgo.MessageCreate) error
```

Comment:
why not directly passing `discordgo` itself ?

This seems to restrain the access of an action to specific part of the api, it that's intended maybe having a bridge between `discordgo` and `actions` might be a good idea

Reply:
The callback that triggers the execution is:
```golang
MessageCreate(s *discordgo.Session, m *discordgo.MessageCreate)
```
These two parameters provide enough accesses and information regarding the context of the call to fully use the discord API. In this case we can consider them as the bridge.